w3af/core/ui/api/main.py

```diff
@@ -43,8 +45,13 @@ def main():
 
     # And finally start the app:
     try:
-        app.run(host=app.config['HOST'], port=app.config['PORT'],
-                debug=args.verbose, use_reloader=False, threaded=True)
+
+        if args.disableSSL:
+            app.run(host=app.config['HOST'], port=app.config['PORT'],
+                    debug=args.verbose, use_reloader=False, threaded=True)
+        else:
+            app.run(host=app.config['HOST'], port=app.config['PORT'],
+                    debug=args.verbose, use_reloader=False, threaded=True, ssl_context=SSLCertifiacate().get())
```

Comment:
Improvement: Call `get(app.config['HOST'])` and change `get` to forward the host to `generate` so the user provided host is used in the SSL cert creation.